packages/css/src/components/avatar/avatar.scss

```diff
@@ -15,6 +15,10 @@
   padding-inline: var(--ams-avatar-padding-inline);
   place-content: center;
 
+  .ams-image {
```

Comment:
You wrote: 

> I decided against using a clean HTML img element because we still want to apply the other base styles of our image component.

Do we? All the aspect ratio stuff in there is irrelevant, because we reset it here. The phantom whitespace thing is not an issue, because this image is a child of a wrapper with an aspect ratio. And the alt text of this image is an empty string. Not much left I think?

Reply:
Alright, let’s just use an HTML `img` element, then.